extensions/localserver-cli-command/package.json

```diff
@@ -18,14 +18,13 @@
 	"main": "index.js",
 	"preferGlobal": true,
 	"dependencies": {
-		"async": "3.0.1",
-		"cors": "2.8.5",
+		"async": "3.2.0",
 		"express": "4.17.1",
 		"glob": "7.1.4",
```

Comment:
Could you upload glob to last version 7.1.6? I have already filled the OSSR (we are using it in other package) and it has been approved already